saas/models.py

```diff
@@ -3384,12 +3385,13 @@ def new_use_charge(self, subscription, use_charge, quantity,
             # Minimum quantity for a use charge is one.
             quantity = 1
         amount = use_charge.use_amount * quantity
+        event_id = "sub_%d_%d" % (subscription.id, use_charge.id)
```

Comment:
that makes sense, on the other note -- this is the first time we use string as event_id, in all other cases an id of object is passed. So maybe it makes sense to add an intermediary model meanwhile, which would be associated with potential event sources?

Reply:
Right now it is a bit clunky because we use both integer `pk`s and strings as `event_id`s. Strings were introduced after it was clear some other form of events (ex: coupons) needed to be used. All `event_id` should be rationalized to strings with a prefix that can be used to generate reports along that "event".